modules/gitops-release-notes-1-13-3.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-13-3_{context}"]
+= Release notes for {gitops-title} 1.13.3
+
+{gitops-title} 1.13.3 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-13.3_{context}"]
+== Errata updates
+
+[id="rhsa-XXXX:NNNN-gitops-1-13-3-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.13.3 security update advisory
+
+Issued: 2024-11-11
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-13-3_{context}"]
+== Fixed issues
+
+* Before this update, when using the dynamic scaling feature, application controller pods consumed high memory and clusters were not distributed properly across shards. This update ensures that clusters are distributed across shards. link:https://issues.redhat.com/browse/GITOPS-3465[GITOPS-3465]
```

Comment:
@Dhruv-Soni11 please update the JIRA number to https://issues.redhat.com/browse/GITOPS-5811

Reply:
Fixed.